query-node/mappings/src/membership.ts

```diff
@@ -88,6 +92,77 @@ async function getOrCreateMembershipSnapshot({ store, event }: EventContext & St
       })
 }
 
+async function saveMembershipExternalResources(
+  store: DatabaseManager,
+  from: MembershipExternalResource[] | undefined,
+  txExtResources: MembershipMetadata.IExternalResource[] | undefined | null,
+  memberMetadata: MemberMetadata
+): Promise<MembershipExternalResource[] | undefined> {
+  if (!from && !txExtResources) {
+    return
+  }
+
+  for (const prevResource of memberMetadata.externalResources ?? []) {
+    await store.remove(prevResource)
+  }
+
+  const externalResources = (from ?? txExtResources?.flatMap(asMembershipExternalResource) ?? []).map(
+    ({ type, value }) => new MembershipExternalResource({ type, value, memberMetadata })
+  )
+  for (const resource of externalResources) {
+    await store.save<MembershipExternalResource>(new MembershipExternalResource(resource))
```

Comment:
Why is here constructor being called agian, since resources were just constructed at [line](https://github.com/Joystream/joystream/pull/4178/files#diff-9af242945476363410a798a87c9d22b687aeeea210b9b1bbc4d0697fde116b04R110)

Reply:
Right I forgot to remove this one!